.github/workflows/release-old-demo.yaml

```diff
@@ -0,0 +1,140 @@
+###
+#
+# Release the old demo app whenever it's package json changes on a release branch
+#
+###
+
+name: Release old demo App to Netlify
+
+on:
+  push:
+    branches:
+      - release/*
+    paths:
+      - 'packages/demo/package.json'
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  # Reads the version from the current branch
+  get-version:
+    name: Get netlify site id for this version
+    runs-on: ubuntu-latest
+    outputs:
+      branch_version: ${{steps.branch.outputs.branch_version}}
+      site_id: ${{steps.branch.outputs.site_id}}
+
+    steps:
+      - name: Read version from branch name
+        id: branch
+        run: |
+          ref=${{github.ref}}
+          branch_version="${ref##*/}"
+          site_id=swisspost-design-system-${branch_version}
+          echo "site_id=$site_id" >> $GITHUB_OUTPUT
+          echo "branch_version=$branch_version" >> $GITHUB_OUTPUT
+
+  # Tries to create a new netlify site. If the site with this version already exists, the job will fail
+  create-site:
+    name: Create a new netlify site
+    runs-on: ubuntu-latest
+    needs: [get-version]
+
+    steps:
+      - name: Create netlify sites
+        env:
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+        run: netlify sites:create --name "${{needs.get-version.outputs.branch_version}}" --account-slug "${{secrets.NETLIFY_ACCOUNT_SLUG}}"
+
+  # Update redirect file through a PR
+  update-redirect:
+    name: Update the _redirects file
+    runs-on: ubuntu-latest
+    needs: [create-site, get-version]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Find _redirects file
```

Comment:
Is this file comming from netlify or what?
Could be nice to write some line about it here.

Reply:
It's a settings file for netlify.